src/Kros.ApplicationInsights.Extensions/ApplicationInsightsExtension.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Kros.ApplicationInsights.Extensions.Options;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Kros.ApplicationInsights.Extensions
```

Comment:
By my opinion `Microsoft.Extensions.DependencyInjection` will be better namespace.